tests/keyboard/scancode/test_unicode_to_scancode.cpp

```diff
@@ -0,0 +1,301 @@
+#include "test_only/test_framework/redemption_unit_tests.hpp"
+
+#include "keyboard/scancode/unicode_to_scancode.hpp"
+
+namespace
+{
+    using namespace scancode;
+
+    Scancode16bits_t get_and_pop_elem(ScancodeSeq_t& scancode_seq)
+    {
+        Scancode16bits_t scancode = scancode_seq.front();
+
+        scancode_seq.pop();
+        return scancode;
+    }
+
+
+    class FrenchLocaleFixture
+    {
+      protected :
+        static constexpr int _LCID = 0x0000040c;
```

Comment:
`_` follwed by uppercase letter are reserved by the standard: https://en.cppreference.com/w/cpp/language/identifiers

Reply:
So, is it ok to do "LCID_" ?